dual_net.py

```diff
@@ -54,12 +53,21 @@
 flags.DEFINE_integer('trunk_layers', go.N,
                      'The number of resnet layers in the shared trunk.')
 
+flags.DEFINE_multi_integer('lr_boundaries', [10000, 30000],
+                     'The number of steps at which the learning rate will decay')
+
+flags.DEFINE_multi_float('lr_rates', [2e-2, 1e-3, 1e-4],
+                     'The different learning rates')
+
 flags.DEFINE_float('l2_strength', 1e-4,
                    'The L2 regularization parameter applied to weights.')
 
 flags.DEFINE_float('sgd_momentum', 0.9,
                    'Momentum parameter for learning rate.')
 
+flags.DEFINE_string('model_dir', None,
```

Comment:
hey @brilee i'm not real psyched about adding a global 'model_dir' flag (which would then be superfluous with a bunch of main.py commands, no?), but i'm not sure what a better way to plumb it into the host_call_fn is.  Ideas?

Reply:
Another thing that's kind of weird about using this as a flag is that anything importing dual_net (selfplay, etc.) then gets this flag defined. It's sort of harmless in that it's not required, but still makes me feel funny.